src/core/evaluator.js

```diff
@@ -1324,6 +1365,16 @@ class PartialEvaluator {
         id: optionalContent.objId,
       };
     } else if (optionalContentType === "OCMD") {
+      const expression = optionalContent.get("VE");
+      if (Array.isArray(expression)) {
+        const result = [];
+        this._parseVisibilityExpression(expression, 0, result);
```

Comment:
Is there ever a risk, in a badly generated PDF document, that the `result` array is *empty* here?
If that were to happen: is it correct to use the empty array here, or should we fallback to the "regular" parsing below?

Reply:
It's probably safer to return the result only if its length > 0